### ISSUE TITLE
Packaging existing binaries for multiple platforms

### ISSUE THREAD

**Issue by kevswims** [shreyasbharath/conan-test-package#3]
I have a set of binaries that are compiled for multiple platforms. ex: folder for Win32, Win64, Linux32 etc. I have been able to create a conan package for the Win64 binaries that works correctly. However, when using the Win32 binaries I cannot figure out how to specify the configuration setting to generate a different has for the Win32 configuration. How is this supposed to be defined in the package recipe or otherwise?

I have tried modifying the arch setting in the  conaninfo.txt but this still produced the same hash as the Win64 package.

**Comment by shreyasbharath** [comment]
@memsharded Can you help with this one?

**Comment by kevswims** [comment]
This is mostly resolved by this question I asked on StackOverflow: https://stackoverflow.com/questions/45202046/using-conan-to-package-multiple-configurations-of-preexisting-binaries

I think the documentation of this example should be updated to include the info given in the response. This is the line that should be mentioned: 
`$ conan package_files libname/1.1.1@name/testing -s arch=x86  # 32 bits instead of 64`

The answer's explanation of profiles is not clear to me either but that is beyond the scope of this issue.

**Comment by shreyasbharath** [comment]
That's great! Can you please create a PR with the changes?

**Comment by kevswims** [comment]
Yes, I'll try and get a pull request submitted by the end of the weekend.

**Comment by memsharded** [comment]
Ok, great, if @kevswims can help with this, really appreciated (sorry for the delay).

I also think that the example of consumer is a bit off-topic and also  not the standard. Packages created with this technique are just regular packages, and can be consumed exactly the same as any other conan package, so not necessary.

I will also try to contribute when @kevswims does the PR. Thanks to both!

**Comment by shreyasbharath** [comment]
@memsharded Can you look at the changes? I am also planning to test packaging out with a custom non-default architecture so can improve the documentation once I play around with it.

**Comment by kevswims** [comment]
Let me know if you need any more help with this use case. I ended up writing an automated script that runs through a set of directories containing binaries for multiple platforms, packages them, uses a set of profiles for each platform for the settings, and uploads them to an artifactory instance.

**Comment by shreyasbharath** [comment]
Ah awesome! Is the recipe something you can share? Maybe you can create another repo showcasing that? It would be really useful because I am just about to embark on that same journey!
